mesonbuild/scripts/meson_install.py

```diff
@@ -168,6 +169,48 @@ def install_man(d):
         else:
             do_copyfile(full_source_filename, outfilename)
 
+def install_fs_links(d):
+    for source, to, type_ in d.fs_links:
+        to += os.path.splitext(source)[1]
```

Comment:
it's going to do the wrong thing. I can fix that assuming that we decide that this is worth pursuing.